ci-operator/step-registry/openshift/manifests/external/openshift-manifests-external-commands.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+LOCALPATH="${SHARED_DIR}/manifest_external.yaml"
+echo "${SHA256_HASH} -" > /tmp/sum.txt
+if ! curl -fLs "${URL}" | tee "${LOCALPATH}" | sha256sum -c /tmp/sum.txt >/dev/null 2>/dev/null; then
+  echo "Expected file at ${URL} to have checksum ${SHA256_HASH} but instead got $(< "${LOCALPATH}" sha256sum | cut -d' ' -f1)"
+  exit 1
+fi
+echo "Downloaded ${URL}, sha256 checksum matches ${SHA256_HASH}"
+
+# Check file syntax
+pip3 install pyyaml --user
+python3 -c 'import yaml;
```

Comment:
Semicolons are not needed, [proof](https://github.com/openshift/release/blob/175b14b7428103fb56f9e5fd01a3380096fbf832/ci-operator/step-registry/single-node/conf/aws/single-node-conf-aws-commands.sh#L13)